data-source/jsonnet/england-wales/ccs/blocks/individual/confirm_dob.jsonnet

```diff
@@ -49,12 +49,12 @@ local nonProxyTitle = {
   ],
 };
 local nonProxyYesLabel = {
-  text: 'Yes, I am {age_in_years} years old',
+  text: 'Yes',
```

Comment:
Placeholder is no longer used so this variable can be removed and just added to the structure directly.